service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/RedisPolicyEvaluationCache.java

```diff
@@ -123,21 +97,7 @@ public void setCachedEvalTimeToLiveSeconds(final long cachedEvalTimeToLiveSecond
     }
 
     @Override
-    void setResourceTranslation(final String fromKey, final String toKey) {
-        this.redisTemplate.opsForSet().add(fromKey, toKey);
-    }
-
-    @Override
-    void setResourceTranslations(final Set<String> fromKeys, final String toKey) {
-        this.redisTemplate.execute(new RedisCallback<List<Object>>() {
-            @Override
-            public List<Object> doInRedis(final RedisConnection connection) throws DataAccessException {
-                StringRedisConnection stringRedisConn = new DefaultStringRedisConnection(connection);
-                for (String fromKey : fromKeys) {
-                    stringRedisConn.sAdd(fromKey, toKey);
-                }
-                return null;
-            }
-        });
+    void setIfNotExists(final String key, final String value) {
+        this.redisTemplate.boundValueOps(key).setIfAbsent(value);
```

Comment:
Timeout should not be set for anything other then Policy Evaluation. This is because we would then be unnecessarily reevaluating requests when entities have no changed.